playbooks/roles/streisand-gateway/tasks/main.yml

```diff
@@ -70,6 +70,13 @@
     - "{{ nginx_private_key }}"
     - "{{ streisand_gateway_password_file }}"
 
+- name: Copy the streisand gateway password file locally
```

Comment:
I'll put this behind the vagrant only var as well. No point doing it except for Vagrant (so far) since it's embedded into the generated docs index for human consumption.